{ 'cinder': { 'left_arm': [ { 'duration': 5.004,
                              'name': u'cinder_0',
                              'positions': [ 0.0676,
                                             -1.5318,
                                             -0.0063,
                                             0.0018,
                                             -1.579,
                                             -0.0001,
                                             0.0001],
                              'starttime': 0.0},
                            { 'duration': 2.887,
                              'name': u'cinder_1',
                              'positions': [ 0.2979,
                                             -3.0175,
                                             -0.0056,
                                             -0.0738,
                                             -1.5691,
                                             -0.0501,
                                             -0.0],
                              'starttime': 10.125},
                            { 'duration': 2.998,
                              'name': u'cinder_2',
                              'positions': [ 0.2526,
                                             -2.4947,
                                             -0.0089,
                                             -0.0734,
                                             -1.5682,
                                             -0.0525,
                                             0.0003],
                              'starttime': 13.012}],
              'left_leg': [ { 'duration': 10.088,
                              'name': u'stand_up_5_left',
                              'positions': [ 0.0,
                                             -0.2557,
                                             -1.57,
                                             3.1,
                                             1.56,
                                             -0.2537],
                              'starttime': 0.0},
                            { 'duration': 2.998,
                              'name': u'cinder_1',
                              'positions': [ 0.0294,
                                             -0.253,
                                             -0.1572,
                                             1.8041,
                                             1.5805,
                                             -0.26],
                              'starttime': 13.021}],
              'head': [ { 'duration': 1.0,
                          'name': u'cinder_0',
                          'positions': [-0.0, -1.5029],
                          'starttime': 0.0}],
              'right_arm': [ { 'duration': 5.004,
                               'name': u'cinder_0',
                               'positions': [ -0.0676,
                                              1.5318,
                                              0.0063,
                                              -0.0018,
                                              1.579,
                                              0.0001,
                                              -0.0001],
                               'starttime': 0.0},
                             { 'duration': 2.887,
                               'name': u'cinder_1',
                               'positions': [ -0.2979,
                                              3.0175,
                                              0.0056,
                                              0.0738,
                                              1.5691,
                                              0.0501,
                                              0.0],
                               'starttime': 10.125},
                             { 'duration': 2.998,
                               'name': u'cinder_2',
                               'positions': [ -0.2526,
                                              2.4947,
                                              0.0089,
                                              0.0734,
                                              1.5682,
                                              0.0525,
                                              -0.0003],
                               'starttime': 13.012}],
              'right_leg': [ { 'duration': 10.088,
                               'name': u'stand_up_5_left',
                               'positions': [ -0.0,
                                              0.2557,
                                              1.57,
                                              -3.1,
                                              -1.56,
                                              0.2537],
                               'starttime': 0.0},
                             { 'duration': 2.998,
                               'name': u'cinder_1',
                               'positions': [ -0.0294,
                                              0.253,
                                              0.1572,
                                              -1.8041,
                                              -1.5805,
                                              0.26],
                               'starttime': 13.021}],
              'torso': [ { 'duration': 2.887,
                           'name': u'cinder_1',
                           'positions': [-0.0025, 1.4279],
                           'starttime': 10.112}]}}
